site/blog/2016/jul/1/sphinx-and-rtd-for-writers.rst

```diff
@@ -0,0 +1,552 @@
+An introduction to Sphinx and Read the Docs for Technical Writers
+=================================================================
+
+By: Eric Holscher
+
+Bio: Eric Holscher is the co-founder of Read the Docs, and the Write the Docs conference.
+Along with community organizing and open source work,
+he does consulting around software documentation and speaks at a number of industry events each year.
+When he isn't expressing his views on software docs, he's getting views from the top of mountains.
+
+Treating documentation as code is becoming a major theme in the software industry.
+This is coming from both sides,
+with developers starting treat documentation as a priority alongside tests and code,
+and writers seeing a lot of value in integrating more into the development process.
+This marrying of cultures isn't simple,
+but having the proper tools for the job makes both sides happy with the results that get produced.
```

Comment:
A subtle argument to be made here is that having the correct tools doesn't make me happier with the output, I'm happier knowing the processes around authoring is frictionless. There's definitely an argument for the output being of a high quality, but as someone investing my time in the output, I'm effected more by the (in)efficiencies of the process.